vms/avm/camino_service_test.go

```diff
@@ -13,10 +13,8 @@ import (
 
 func TestGetAssetDescriptionC4T(t *testing.T) {
 	env := setup(t, &envConfig{})
-	defer func() {
-		require.NoError(t, env.vm.Shutdown(context.Background()))
-		env.vm.ctx.Lock.Unlock()
-	}()
+	env.vm.ctx.Lock.Unlock()
+	defer stopEnv(t, env)
```

Comment:
`setup` call Lock and expects caller to call`Unlock`
other avax service tests are enveloping shutdown with lock-unlock, so we're doing the same